cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -3400,9 +3401,14 @@ def _test_api_v2_tasks_id_data_spec(self, user, spec, data,
         self.assertEqual(response.status_code, status.HTTP_202_ACCEPTED, response.reason_phrase)
 
         if get_status_callback:
-            response = get_status_callback(task_id, user)
-            while response.data['state'] not in ('Failed', 'Finished'):
+            max_number_of_attempt = 10
```

Comment:
0.01 * 10 = 0.1 seconds, it's quite a small time period even in some normal situations, e.g. with video processing and chunks on weak machines. I suggest increasing it to 10 seconds instead for stability.